src/Native/Windows/CMakeLists.txt

```diff
@@ -81,6 +82,18 @@ else()
     set(DYNAMIC_UCRT_LIB "ucrtd.lib")
 endif()
 
+# App-Container builds require a more specific linking to ensure the APIs imported pass WACK
+if ($ENV{__appContainer} STREQUAL true AND UPPERCASE_CMAKE_BUILD_TYPE STREQUAL "RELEASE")
+    set(__LinkCustomCRT true)
+    set(__LinkArgs "${__LinkArgs} /LTCG")
```

Comment:
Why is this enabled only when the AppContainer bit is set?  Our release builds should always enable link-time code generation (/GL to the compiler and /LTCG to the linker).